account_financial_report/wizard/general_ledger_wizard.py

```diff
@@ -74,8 +74,9 @@ class GeneralLedgerReportWizard(models.TransientModel):
         help="Ending account in a range",
     )
     grouped_by = fields.Selection(
-        selection=[("", "None"), ("partners", "Partners"), ("taxes", "Taxes")],
+        selection=[("none", "None"), ("partners", "Partners"), ("taxes", "Taxes")],
```

Comment:
I think changing this value will provoke side effects when checking for falsy values across the code. Have you checked it?

Reply:
I see your point, ignore my comment:
the problem is that this field will be always set (as it is required) and then always "truthy" even if set to "none".

Reply:
That's why I don't feel comfortable changing this on this old stable version. Other dependent modules may fail because they check `not record.grouped_by`, and since this, they logic will change.